feat(admin): group synchronization dialog now displays the interval b…

xflord opened the pull request:
…etween synchronizations

bodnara reviewed (changes requested):
The interval is displayed properly, well done on that. However this dialog seems to be quite old and looks inconsistent with the reset of the app. I would like to ask you if you would be so kind and implement following changes:
![image](https://user-images.githubusercontent.com/23495969/201603437-8366d669-7cdd-4654-812c-27ff4870b976.png)

- Change **all** the rows to be a input mat-form-field as shown in the screenshot (readonly), also remove ":" from the placeholder
- Change the order and color of buttons as per screenshot
- Remove redundant methods/elements that are in this component

mattjoke commented on libs/perun/dialogs/src/lib/group-sync-detail-dialog/group-sync-detail-dialog.component.ts:
```diff
@@ -96,26 +83,6 @@ export class GroupSyncDetailDialogComponent implements OnInit {
     return this.type === 'STRUCTURED';
   }
 
```
I think `isBasic` and `isStructured` methods are not used. I think you should remove them

mattjoke reviewed (commented):
Otherwise I think it looks good.

bodnara commented on apps/admin-gui/src/app/vos/pages/group-detail-page/group-detail-page.component.html:
```diff
@@ -43,16 +43,12 @@
-                *ngIf="syncAuth"
-                [disabled]="(group | groupSyncIcon) === 'sync_disabled'"
-                mat-icon-button>
+              <button (click)="onSyncDetail()" mat-icon-button>
```
Here we lost the information if user is authorized to force group synchornization. We can move this check to the dialog itself so we can still open it but cannot perform the action.

bodnara reviewed (changes requested):
There are couple of improvements:
![image](https://user-images.githubusercontent.com/23495969/201921009-8e7b8b8c-43d5-45a4-8994-2c1fe560dc94.png)

- The synchronization field is not translated
- The dialog width is not set up, please set some reasonable width (450px maybe) when opening the dialog
- Right now the logic of sync and syncStructured is duplicated (2 buttons, 2 methods), I guess just merge it to one
- Last thing, I know the last sync time has the same format as before, but maybe we could use something more readable (this point more of a reference thing, if you think the format is fine than leave it as is)

xflord commented:
> Last thing, I know the last sync time has the same format as before, but maybe we could use something more readable (this point is more of a preference thing, if you think the format is fine than leave it as is)

For now, I just removed the milliseconds and we can discuss it further tomorrow.
Fixed the rest

HejdaJakub commented on libs/perun/dialogs/src/lib/group-sync-detail-dialog/group-sync-detail-dialog.component.ts:
```diff
@@ -41,33 +40,21 @@ export class GroupSyncDetailDialogComponent implements OnInit {
   onForce(): void {
     this.loading = true;
-    if (this.isBasic()) {
+    if (this.type === 'BASIC') {
```
Again please replace the deprecated subscription by the recommended observer argument `{next: (), error: ()}`

HejdaJakub reviewed (commented):
Just one suggestion, it looks good otherwise.

mattjoke reviewed (approved):
Looks good :+1:
